Spigot-Server-Patches/0544-Create-Player-sendActionBar-with-component-support.patch

```diff
@@ -0,0 +1,34 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Tuck <jan@tuck.dk>
+Date: Fri, 17 Jul 2020 01:54:20 +0200
+Subject: [PATCH] Create Player#sendActionBar with component support
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index b832ece4948323e60fabecdeee5c9051b65c5e8e..4e05437d3ca725732a7d015f2a71db5a9d962439 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -245,8 +245,7 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+     // Paper start
+     @Override
+     public void sendActionBar(String message) {
+-        if (getHandle().playerConnection == null || message == null || message.isEmpty()) return;
+-        getHandle().playerConnection.sendPacket(new PacketPlayOutChat(new net.minecraft.server.ChatComponentText(message), net.minecraft.server.ChatMessageType.GAME_INFO, SystemUtils.getNullUUID()));
++        sendActionBar(net.md_5.bungee.api.chat.TextComponent.fromLegacyText(message));
+     }
+ 
+     @Override
+@@ -255,6 +254,13 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+         sendActionBar(org.bukkit.ChatColor.translateAlternateColorCodes(alternateChar, message));
+     }
+ 
++    @Override
++    public void sendActionBar(BaseComponent... components) {
++        if (getHandle().playerConnection == null || components == null || components.length == 0) return;
++        PacketPlayOutChat packetPlayOutChat = new PacketPlayOutChat(null, net.minecraft.server.ChatMessageType.GAME_INFO, SystemUtils.getNullUUID());
```

Comment:
Why do you send `null` instead of the supplied components?

Reply:
Supplying null was correct, but I forgot to set the components in the pr I sent, thank you for noticing.